pymc/data.py

```diff
@@ -12,34 +12,31 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 
-import collections
 import io
 import os
 import pkgutil
 import urllib.request
 import warnings
 
 from copy import copy
-from typing import Any, Dict, List, Optional, Sequence, Tuple, Union, cast
+from typing import Dict, Optional, Sequence, Tuple, Union, cast
 
 import aesara
 import aesara.tensor as at
 import numpy as np
 
 from aesara.compile.sharedvalue import SharedVariable
-from aesara.graph.basic import Apply
 from aesara.tensor.type import TensorType
 from aesara.tensor.var import TensorConstant, TensorVariable
 
 import pymc as pm
 
-from pymc.aesaraf import convert_observed_data
+from pymc.aesaraf import at_rng, convert_observed_data
```

Comment:
We can also get rid of `at_rng`, as the seeds created by it will be overwritten in `compile_pymc` anyway. Just import `RandomStream` wherever it's needed.

Reply:
ok